app/src/main/java/com/google/android/samples/socialite/ui/chat/ChatScreen.kt

```diff
@@ -500,27 +541,34 @@ private fun PreviewInputBar() {
     }
 }
 
+@OptIn(ExperimentalSharedTransitionApi::class)
 @Preview
 @Composable
 private fun PreviewChatContent() {
-    SocialTheme {
-        ChatContent(
-            chat = ChatDetail(ChatWithLastMessage(0L), listOf(Contact.CONTACTS[0])),
-            messages = listOf(
-                ChatMessage("Hi!", null, null, 0L, false, null),
-                ChatMessage("Hello", null, null, 0L, true, null),
-                ChatMessage("world", null, null, 0L, true, null),
-                ChatMessage("!", null, null, 0L, true, null),
-                ChatMessage("Hello, world!", null, null, 0L, true, null),
-            ),
-            input = "Hello",
-            sendEnabled = true,
-            onBackPressed = {},
-            onInputChanged = {},
-            onSendClick = {},
-            onCameraClick = {},
-            onPhotoPickerClick = {},
-            onVideoClick = {},
-        )
+    SharedTransitionScope {
+        AnimatedContent(targetState = 1) {_ ->
```

Comment:
Rather use AnimatedVisibility(true) here instead of AnimatedContent for previews.